timemachine/fe/atom_mapping.py

```diff
@@ -44,6 +46,51 @@
 # - refinement of marcs matrix is done on uint8 arrays
 
 
+def get_cores_and_total_nodes_visited(
```

Comment:
We should add a basic test of this method, to at least to say `total_nodes_visited > 0`

Reply:
Thinking about it some more, if there is a case that we could test that when you change the number of valid core sizes the number changes? But might be flaky if we make further changes. Just me spit balling, most important is just to test that this is returning some sane value.

Reply:
Added a basic test in https://github.com/proteneer/timemachine/pull/1231/commits/59e3e0267cb70b3232758605be68e0c7f89e1ddb.

> One test I thought of would be to set max_nodes=1 and assert that total_nodes_visited > 1

This makes sense to me as long as the mols are not perfectly mapped (in which case total_nodes_visited would equal 1?). Will add this case.

Reply:
The test you added is sufficient for me, my suggestion seems janky. But never say no to more tests lol

Reply:
Actually a bit confused here. Setting `max_nodes=1` causes the MCS search to fail with `NoMappingError`.

Reply:
It looks like it has to traverse at least the number of atoms in mol a??? https://github.com/proteneer/timemachine/blob/master/timemachine/fe/mcgregor.py#L293-L295

So `max_nodes` has to be at least `mol_a.GetNumAtoms()`?